### DISCUSSION TITLE
https://blog.3place.cn/3place/sszs/SSZS006.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#107] [General]
# https://blog.3place.cn/3place/sszs/SSZS006.html

https://blog.3place.cn/3place/sszs/SSZS006.html

**Comment by picklecai** [comment]
姚弘业先生，这也是一位谭嗣同呀！

**Comment by picklecai** [comment]
没想到律诗是这样理解的
